components/card/test/card.visual-diff.js

```diff
@@ -33,7 +33,13 @@ describe('d2l-card', () => {
 		{ name: 'no-link-focus', selector: '#header-content', action: (selector) => page.$eval(selector, (elem) => elem.focus()) },
 		{ name: 'subtle', selector: '#subtle' },
 		{ name: 'link', selector: '#link' },
-		{ name: 'link-focus', selector: '#link', action: (selector) => page.$eval(selector, (elem) => elem.focus()) },
+		{ name: 'link-focus', selector: '#link', action: (selector) => page.$eval(selector, (elem) => {
+			elem.addEventListener('blur', () => {
+				elem.classList.remove('focus-visible');
+			}, { once: true });
+			elem.classList.add('focus-visible');
+			elem.focus();
+		}) },
```

Comment:
There's a `forceFocusVisible` helper that does exactly this (I think you might have copied this from there?). To use it from visual diff tests is a bit tedious though as you have to [import it from the HTML file](https://github.com/BrightspaceUI/core/blob/master/components/button/test/button-icon.visual-diff.html#L9) and expose it as a global on the window.

Reply:
In this case since card itself isn't focusable the `forceFocusVisible` method attempts to find a focusable child which is the `a` tag within the card. The method adds the `focus-visible` class to that instead of the host element which causes the styles to not appear properly

Reply:
Interesting, and it looks like the polyfill puts `.focus-visible` on both the host AND the `<a>` which I wasn't quite expecting. 👍